content/guides/asset-management.md

```diff
@@ -591,7 +591,7 @@ __webpack.config.js__
 __src/index.js__
 
 ``` diff
-- import _ from 'lodash';
+import _ from 'lodash';
```

Comment:
Good catch! Please replace `- ` with `  ` to keep it consistent with other unchanged lines in the file.